op-challenger/game/keccak/types/types.go

```diff
@@ -22,6 +25,10 @@ type Leaf struct {
 	StateCommitment common.Hash
 }
 
+// Proof is a list of [common.Hash]s that prove the merkle inclusion of a leaf.
+// These are the sibling hashes of the leaf's path from the root to the leaf.
+type Proof [BinaryMerkleTreeDepth]common.Hash
```

Comment:
Unless there was a cycle formed somewhere, I'd keep these in the `merkle` package - they're quite merkle specific types.

Reply:
Ah that's right, there was a cycle dependency earlier, but that since has been resolved after cleaning up. Fixed!